tools/ci/testrun/script/test_example/test_example.py

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/python3
+# encoding: utf-8
+import pytest
+
+pytestmark = [pytest.mark.common, pytest.mark.qemu]
+do_not_support = ["sabre-6quad", "rv-virt", "rv-virt64", "esp32c3-devkit", "bl602evb"]
+
+
+def test_hello(p):
+    ret = p.sendCommand("hello", "Hello, World!!")
+    assert ret == 0
+
+
+def test_helloxx(p):
+    if p.board in do_not_support:
+        pytest.skip("unsupported at {}".format(p.board))
+    if p.board == "best1600_ep":
+        if p.core == "tee":
+            pytest.skip("unsupported at {}-{}".format(p.board, p.core))
+    ret = p.sendCommand("helloxx", "Hello, World!!")
```

Comment:
I think the "helloxx" command doesn't return "Hello, World!!"! Did you test it?
It should return many different messages depending on which C++ features is enabled.
And finally it should return:
```
HelloWorld: mSecret=42
```

Reply:
I got the result like below, looks like different with yours. Image build by citest deconfig. @acassis 

nsh> helloxx
helloxx_main: Saying hello from the dynamically constructed instance
CHelloWorld::HelloWorld: Hello, World!!
helloxx_main: Saying hello from the instance constructed on the stack
CHelloWorld::HelloWorld: Hello, World!!
helloxx_main: Saying hello from the statically constructed instance
CHelloWorld::HelloWorld: Hello, World!!

Reply:
@acassis both will print to console:
https://github.com/apache/incubator-nuttx-apps/blob/master/examples/helloxx/helloxx_main.cxx#L67-L78
so it's fine to check "Hello, World!!".